Makefile

```diff
@@ -213,9 +215,17 @@ test: assets
 	$(PYTHON) \
 		-m pytest $(PYTEST_ARGS) --durations=10\
 		--ignore-glob="$(TESTDIR)/**/*bench*.py" \
+		--ignore-glob="$(TESTDIR)/network/*.py" \
 		$(TESTDIR)
 	cd ocrd_utils ; $(PYTHON) -m pytest --continue-on-collection-errors -k TestLogging -k TestDecorators $(TESTDIR)
 
+INTEGRATION_TEST_IN_DOCKER = docker exec core_test
+integration-test:
+	$(DOCKER_COMPOSE) --file tests/network/docker-compose.yml up -d
+	sleep 20
+	$(INTEGRATION_TEST_IN_DOCKER) pytest -k 'test_rmq or test_db or test_processing_server' -v
+	$(DOCKER_COMPOSE) --file tests/network/docker-compose.yml down --remove-orphans
+
```

Comment:
Some tests are failing currently on my machine: `test_db_workspace_update` (fails only sometimes), `test_processing_server_processing_request` and `test_processing_server_workflow_request`. 
When the tests are failing, the containers started by the Makefile are not stopped afterwards. I still have to find out why the tests fail. I will comment when I know more

Reply:
Unfortunately, I have not found a way yet to stop and remove the containers when some tests fail. What I use currently is manually stopping and removing all containers with 2 aliases for:
```bash
docker stop $(docker ps --format="{{.ID}}")
docker rm $(docker ps -a --format="{{.ID}}")
```

The reason why some DB tests fail is that you executed the tests again without stopping the DB container. I did not implement dropping DB collections when re-executing tests. Hence, some tests fail when entries exist in the DB. In case you know how to achieve dropping DB collections simply without refactoring the Processing Server and without introducing a second MongoDB client - feel free to commit.

Another reason why some Processing Server tests may fail is because the socket files for the mets server are still preserved in the host machine. Make sure no sockets are left from previous test executions.

Reply:
This is now fully resolved. The socket files left from previous test executions are deleted before the tests.

Reply:
Yes, test are working now